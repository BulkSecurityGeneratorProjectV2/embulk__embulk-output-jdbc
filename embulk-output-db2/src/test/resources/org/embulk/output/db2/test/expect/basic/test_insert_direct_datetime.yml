table: TEST_DATETIME
mode: insert_direct
