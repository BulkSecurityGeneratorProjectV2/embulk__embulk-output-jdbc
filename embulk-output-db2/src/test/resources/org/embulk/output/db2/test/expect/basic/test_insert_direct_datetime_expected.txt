----- ------------ ---------- ------------------------------ ------------------------ -------------------------------------
1001  2016-09-08   12:34:45   2016-09-09 12:34:45.123456     2016-09-10 12:34:45      2016-09-11 12:34:45.123456789000
1002  2016-12-31   23:59:59   2016-12-31 23:59:59.999999     2016-12-31 23:59:59      2016-12-31 23:59:59.999999999000
1003
9999
ID    DATE_ITEM    TIME_ITEM  TIMESTAMP_ITEM                 TIMESTAMP0_ITEM          TIMESTAMP12_ITEM
